lib/forklift/box_distributor.rb

```diff
@@ -134,6 +135,28 @@ def configure_shell(machine, box)
       end
     end
 
+    # Configures synced folders defined for the box
+    # and the private network required for them
+    def configure_synced_folders(machine, box)
+      configure_private_network(machine, box)
+
+      box.fetch('synced_folders', []).each do |folder|
+        machine.vm.synced_folder folder['path'], folder['mount_point'], folder['options'].symbolize_keys
```

Comment:
`with_indifferent_access` is a Rails method, like `symbolize_keys`.

The keys only need to be symbols, since Vagrant expects them for options.
I will move the method into `BoxDistributor`, since that is the only place for now where it is needed.

I just saw that `configure_networks` too needs that, I'll use it there as well then.